src/modules/Search.js

```diff
@@ -72,7 +72,8 @@ module.exports = function (searchService, downloadService, options, client) {
    * @param {array} filesByUser
    */
   this.showResults = (filesByUser) => {
-    log(chalk.green('Displaying search results'));
+	  var numResults = Object.keys(filesByUser).length;
```

Comment:
Can you use `const`instead of `var` please?

Reply:
Also, make sure to use spaces instead of tabs (see [Contributing guide](https://github.com/aeyoll/soulseek-cli/blob/develop/CONTRIBUTING.md))